setup.cfg

```diff
@@ -1,2 +1,7 @@
 [flake8]
 ignore=E501
+
+[pytest]
+addopts=-r=fEsxXw
+base_url=https://www-dev.allizom.org
```

Comment:
It turned out that it was an existing `pytest.ini` that was causing the problem.